README.md

```diff
@@ -205,4 +209,10 @@ export interface ILocationRequestParams {
 
 ---
 
+## Credits
+
+Logo by [Caneco](httsp://www.twitter.com/caneco).
```

Comment:
Typo: httsp should be https 😉

Reply:
Well catched. Thanks 😄

Reply:
Hi @caneco do you want to submit another MR to fix this?